_posts/de/newsletters/2025-01-10-newsletter.md

```diff
@@ -0,0 +1,99 @@
+
+  Roberts Vorschlag sieht ein Verfahren vor, das es einem Kanalteilnehmer erlaubt, nur einen einzigen Status zu bestätigen.  Gemäß der von Roberts vorgeschlagenen Methode ist es einem Kanalteilnehmer möglich, lediglich einen einzigen Status zu bestätigen. Im Falle einer späteren Bestätigung eines weiteren Status durch einen anderen Teilnehmer oder Teilnehmer, der keinen Status übermittelt hat, ist es den zuvor genannten Teilnehmern möglich, das Geld der Teilnehmer zu erhalten, die veraltete Zustände eingereicht haben.
+
+
+
+  Leider wurde nach der Publikation des Schemas ein kritischer Fehler darin festgestellt. Ähnlich wie bei dem in der vorherigen Meldung beschriebenen "_2x delay Problem_", das in der vorherigen Nachricht thematisiert wurde, kann die letzte Partei, die unterschreibt, einen Zustand vervollständigen, den keine andere Partei vervollständigen kann. Dadurch erhält der letzte Unterzeichner exklusiven Zugriff auf den aktuellen Endzustand.  Roberts stellte nach der Veröffentlichung des Schemas einen kritischen Fehler darin fest. Ähnlich wie bei dem in der vorherigen Meldung beschriebenen 2x-Delay-Problem kann die letzte Partei, die unterschreibt, einen Zustand vervollständigen, den keine andere Partei vervollständigen kann. Dadurch erhält der letzte Unterzeichner exklusiven Zugriff auf den aktuellen Endzustand. Wenn eine andere Partei versucht, im vorherigen Zustand abzuschließen, wird sie Geld verlieren, wenn der Endunterzeichner den Endzustand verwendet. Roberts untersucht derzeit alternative Ansätze, aber das Thema hat eine interessante Diskussion darüber ausgelöst, ob die Hinzufügung eines Strafmechanismus zur LN-Symmetrie sinnvoll ist.  Gregory Sanders, dessen frühere Proof-of-Concept-Implementierung von LN-Symmetry ihn zu der Überzeugung brachte, dass ein Strafmechanismus unnötig ist (siehe [Newsletter #284][news284 sympen]), merkte an, dass der "repeated-old-state"-Angriff ähnlich ist wie ein [replacement cycling attack][topic replacement cycling].
```

Comment:
"Gregory Sanders, dessen frühere Proof-of-Concept-Implementierung von LN-Symmetry <s>ihn</s> zu der Überzeugung brachte"

Stolpere über das "ihn", ich glaube das sollte gestrichen werden?